### DISCUSSION TITLE
Cannot Upgrade to Pro

### DISCUSSION THREAD

**Discussion by ziyadashraf** [vercel/community#5367] [Account Issue]
### Can't Upgrade to Pro

I tried to upgrade my team called "GoNative" but I always get the error "Failed to upgrade team. Please try again."

**Comment by marvin-splitt** [comment]
I have the same problem 🙏🏼

**Comment by z4nr34l** [comment]
Duplicate of #5371 - you should be able to upgrade your tier now.

**Reply by didyk** [reply]
@z4nr34l hi! I see the same bug

**Reply by z4nr34l** [reply]
@didyk could you bring more info? Screenahots/Error messages so Vercel team would have more info about that.

**Reply by didyk** [reply]
@z4nr34l I clicked on the button for upgrade personal account and this button just redirect me to the dashboard without any modals/popups
